### PR TITLE
Reset slice and volume defaults when a new image is input to the 3D viewer

### LINKED ISSUE #235: Updating input image should reset orientation of volume render
- [ ]  reset orientation of camera

Also:

- [ ] reset whether slice is visible
- [ ] reset whether volume is visible
- [ ] reset which direction we are slicing X/Y/Z
- [ ] delete all clippings and turn off clipping widget

### LINKED ISSUE #238: Clipping planes not correctly updated when new image is selected in a specific case
If you do:

1. load image 1
2. turn on volume
3. clip
4. turn off clipping
5. turn off volume
7. load image 2 instead of 1
8. turn on volume
9. turn on clipping

then the size of the clipping widget is incorrect.

If you skip step 5 and 8, the size of clipping widget is correct.

### PR DESCRIPTION
**Pull request by lauramurgatroyd**
Closes #235 
Closes #238

### CODE REVIEW

File: Wrappers/Python/ccpi/viewer/CILViewer.py [hunk 1]
```diff
@@ -622,18 +633,41 @@ def startRenderLoop(self):
...
-                    self.planew.SetEnabled(False)
-                    self.volume.GetMapper().RemoveAllClippingPlanes()
-                    self.clipping_plane_initialised = False
-                self.installVolumeRenderActorPipeline()
-                self.volume.VisibilityOn()
-            else:
-                self.volume_render_initialised = False
+            if self.clipping_plane_initialised:
+                self.planew.SetEnabled(False)
+                self.volume.GetMapper().RemoveAllClippingPlanes()
```
Comment (paskino) [inline Wrappers/Python/ccpi/viewer/CILViewer.py:642]: These 2 lines seem to be repeated below. Are they necessary?

**Review by paskino — APPROVED** [review]
Looks good to me. Have a look at my comment

File: Wrappers/Python/ccpi/viewer/CILViewer.py (hunk 1 above)
Comment (Pasarus) [reply Wrappers/Python/ccpi/viewer/CILViewer.py:642]: These first 2 lines seem to clear what is already there, then the next 2 seem to fix another issue with the camera position caused by the clipping plane.

Comment (lauramurgatroyd) [reply Wrappers/Python/ccpi/viewer/CILViewer.py]: They aren't necessary, and have now been deleted.

Comment (lauramurgatroyd) [reply Wrappers/Python/ccpi/viewer/CILViewer.py]: Creating the new clipping plane overrides the old one anyway so there is no point deleting them in these 2 lines.

**Pull request merged**